mediacloud/mediawords/util/web/user_agent/__init__.py

```diff
@@ -723,7 +733,7 @@ def __execute_request(self, requests_prepared_request: requests.PreparedRequest)
 
             requests_response.history = []
 
```

Comment:
`requests.TooManyRedirects` I guess returns a valid response?

Reply:
Yes, it returns the last fetched page. Perl's `LWP::UserAgent` wouldn't report much of an error and just return the last fetched page, so we kind of do it that here too.